templates/zurihac2019.html

```diff
@@ -16,6 +16,7 @@
             <li><a href="/zurihac2019#toggle-nav" class="x">X</a></li>
             <li><a href="/zurihac2019#">Home</a></li>
             <li><a href="/zurihac2019#program">Program</a></li>
+            <li><a href="/zurihac2019#schedule">Schedule</a></li>
```

Comment:
Should we have both Program and Schedule?   This seems a little confusing to me.

Reply:
I'm happy to let you decide. My thinking was that it's good to be able to jump straight to the calendar, since I usually check it 10 times per day. Would you be open to taking Program out?

Reply:
How about we just have "program", but we move the schedule/calendar to the very top of that section?

Reply:
Oh, I like that idea! Lets remove that line from this PR then, and we'll make the change in a separate PR